docs/concepts/fields.md

```diff
@@ -111,6 +127,31 @@ print(user.username)
 The `data` argument will *only* contain the already validated data, based on the [order of model fields](./models.md#field-ordering)
 (the above example would fail if `username` were to be defined before `email`).
 
+## Validate default values
```

Comment:
Moved from below, and improved a bit by mentioning the `validate_default` config value as well.